pkg/querier/queryrange/roundtrip_test.go

```diff
@@ -207,7 +207,7 @@ func TestMetricsTripperware(t *testing.T) {
 	queryCount, queryHandler = counter()
 	h = getQueryAndStatsHandler(queryHandler, statsHandler)
 	_, err = tpw.Wrap(h).Do(ctx, lreq)
-	require.Error(t, err)
+	require.ErrorContains(t, err, "query too large to execute on a single querie")
```

Comment:
Tiniest of nits, triggered my OCD 😛 feel free to ignore

```suggestion
	require.ErrorContains(t, err, "query too large to execute on a single querier")
```

Reply:
:joy: